### ISSUE TITLE
Subcutaneous cords

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9908]
**Preferred term label:**
Subcutaneous cords

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.orthobullets.com/hand/6058/dupuytrens-disease?hideLeftMenu=true "Normal fascial bands become pathologic cords"

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Dupuytren's disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
is this the same as [Dupuytren contracture HP:0005679](https://hpo.jax.org/app/browse/term/HP:0005679)?

**Comment by MickeySegal** [comment]
That was our only use of the term so I changed ours to fit with [Dupuytren contracture HP:0005679](https://hpo.jax.org/app/browse/term/HP:0005679).

**Comment by pnrobinson** [comment]
thanks, closing.
